Tests/test_Uniprot.py

```diff
@@ -33,40 +33,87 @@ def test_uni001(self):
         self.assertEqual(seq_record.id, "Q91G55")
         self.assertEqual(seq_record.name, "043L_IIV6")
         self.assertEqual(seq_record.description, "Uncharacterized protein 043L")
-        self.assertEqual(repr(seq_record.seq), "Seq('MDLINNKLNIEIQKFCLDLEKKYNINYNNLIDLWFNKESTERLIKCEVNLENKI...IPI')")
+        self.assertEqual(
+            repr(seq_record.seq),
+            "Seq('MDLINNKLNIEIQKFCLDLEKKYNINYNNLIDLWFNKESTERLIKCEVNLENKI...IPI')",
+        )
 
         # self.assertEqual(seq_record.accessions, ['Q91G55']) #seq_record.accessions does not exist
         # self.assertEqual(seq_record.organism_classification, ['Eukaryota', 'Metazoa', 'Chordata', 'Craniata', 'Vertebrata', 'Mammalia', 'Eutheria', 'Primates', 'Catarrhini', 'Hominidae', 'Homo'])
         # self.assertEqual(record.seqinfo, (348, 39676, '75818910'))
 
         self.assertEqual(len(seq_record.features), 1)
-        self.assertEqual(repr(seq_record.features[0]), "SeqFeature(FeatureLocation(ExactPosition(0), ExactPosition(116)), type='chain', id='PRO_0000377969')")
+        self.assertEqual(
+            repr(seq_record.features[0]),
+            "SeqFeature(FeatureLocation(ExactPosition(0), ExactPosition(116)), type='chain', id='PRO_0000377969')",
+        )
 
         self.assertEqual(len(seq_record.annotations["references"]), 2)
-        self.assertEqual(seq_record.annotations["references"][0].authors, "Jakob N.J., Mueller K., Bahr U., Darai G.")
-        self.assertEqual(seq_record.annotations["references"][0].title, "Analysis of the first complete DNA sequence of an invertebrate iridovirus: coding strategy of the genome of Chilo iridescent virus.")
-        self.assertEqual(seq_record.annotations["references"][0].journal, "Virology 286:182-196(2001)")
-        self.assertEqual(seq_record.annotations["references"][0].comment, "journal article | 2001 | Scope: NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA] | ")
+        self.assertEqual(
+            seq_record.annotations["references"][0].authors,
+            "Jakob N.J., Mueller K., Bahr U., Darai G.",
+        )
+        self.assertEqual(
+            seq_record.annotations["references"][0].title,
+            "Analysis of the first complete DNA sequence of an invertebrate iridovirus: coding strategy of the genome of Chilo iridescent virus.",
```

Comment:
That string/line is still overly long. If you wanted to, you could split it to fit on a line of 88 characters:

```python
            "Analysis of the first complete DNA sequence of an invertebrate iridovirus:"
            " coding strategy of the genome of Chilo iridescent virus.",
```

I think that just fits - fit not, break a word earlier. Note this is using implicit string concatenation. No need for an explicit plus operator here.